content/documentation/v10/batching.md

```diff
@@ -99,7 +99,7 @@ Here is how you might put this in place:
             @Override
             public CompletionStage<List<Object>> load(List<String> keys) {
                 //
-                // we use supplyAsync() of values here for maximum parellisation
+                // we use supplyAsync() of values here for maximum parallelisation
```

Comment:
@andimarek @jasper-vandemalle A random search across books and articles on computer processing reveals that `parallelization ` is more commonly used term relative to `parallelisation`.

Reply:
Let me know if you are guys are okay with opening another PR to fix it across.

Reply:
@nutharsh that would be great, yes